File: src/main/java/com/example/dao/UserDao.java
package com.example.dao;

import com.example.entity.Student;
import org.apache.ibatis.annotations.Param;

import java.util.List;

/**
 * Created by Will on 2016/8/15 13:16.
 */
public interface UserDao {

    Student queryUserByLogingName(@Param(value = "loginName") String loginName);

    List<Student> queryAllUser();
}

File: src/main/java/com/example/dao/UserDao.kt
package com.example.dao

import com.example.entity.Student
import org.apache.ibatis.annotations.Param

/**
 * Created by zoumy on 2017/5/12 9:34.
 */
interface UserDao {

    fun queryUserByLogingName(@Param(value = "loginName") loginName: String): Student

    fun queryAllUser(): List<Student>
}